src/main/kotlin/com/exactpro/th2/lwdataprovider/configuration/Configuration.kt

```diff
@@ -31,6 +31,8 @@ class CustomConfigurationClass {
     val mode: String? = null
     val grpcBackPressure : Boolean? = null
     val bufferPerQuery: Int? = null
+    val codecUsePinAttributes: Boolean = true
+    val responseFormats: List<String> = emptyList()
```

Comment:
Why do we need those formats in the configuration? Do we specify default formats? If we do the name of the parameter should be changed to `defaultResponseFormats`